jiant/utils/options.py

```diff
@@ -0,0 +1,41 @@
+"""
+Functions to do with parsing configs here.
+"""
+import torch
+
+
+def parse_task_list_arg(task_list):
+    """Parse task list argument into a list of task names."""
+    task_names = []
+    for task_name in task_list.split(","):
+        if task_name == "glue":
+            task_names.extend(ALL_GLUE_TASKS)
+        elif task_name == "superglue":
+            task_names.extend(ALL_SUPERGLUE_TASKS)
+        elif task_name == "none" or task_name == "":
+            continue
+        else:
+            task_names.append(task_name)
+    return task_names
+
+
+def parse_cuda_list_arg(cuda_list):
```

Comment:
structure seems a little odd, though workable. Why not

`elif isinstance(cuda_list, int):
    return cuda_list
elif "," in cuda_list:
    return [int(d) for d in cuda_list.split(",")]
else:
    raise ValueError(...)`